src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -565,23 +556,23 @@ def get_augmented_data(self, images, annotations, random_seed):
             feed_dict = dict()
 
             # Populate feed_dict with images and annotations
-            graph_op = self.resize_op_batch[0:len(images)]
+            graph_op = self.resize_op_batch[0 : len(images)]
             for i in range(len(images)):
                 feed_dict[self.img_tf[i]] = _utils.convert_shared_float_array_to_numpy(
                     images[i]
                 )
-                feed_dict[
-                    self.ann_tf[i]
-                ] = _utils.convert_shared_float_array_to_numpy(annotations[i])
+                feed_dict[self.ann_tf[i]] = _utils.convert_shared_float_array_to_numpy(
+                    annotations[i]
+                )
 
             # Populate feed_dict with random seed and random alpha values, used
             # to sample image perturbations. We don't use TensorFlow's built-in
             # support for random number generation, since we want to effectively
             # reset the seed for each session (batch).
             random = np.random.RandomState(seed=random_seed)
             feed_dict[self.alpha_tf] = random.rand(*self.alpha_tf.shape)
-            feed_dict[self.random_seed_tf] = random.random_integers(
-                0, 2**32 - 1, size=self.batch_size
+            feed_dict[self.random_seed_tf] = random.randint(
```

Comment:
this line

Reply:
randint is inclusive of the endpoints, so we want this to be 0, 2**32-1 to not include 2**32.

Reply:
Ah, numpy.random.randint is different.